Add new General A1 domains we are releasing now (#69154)
Azure/azure-content: articles/cognitive-services/Custom-Vision-Service/select-domain.md

|Domain|Purpose|
|---|---|
|__General__| Optimized for better accuracy with comparable inference time as General Domain. Recommend for larger dataset/harder user scenario. This domain will use more training time. ID: `ee85a74c-405e-4adc-bb47-ffa8ca0c9f31`|

> |__General [A1]__| Optimized for a broad range of image classification tasks. If none of the other domains are appropriate, or you're unsure of which domain to choose, select the General domain. ID: `a8e3c40f-fb4a-466f-832a-5e457ae4a344`|

Reviewer: Is A1 for larger datasets? these descriptions seem to be reversed

Author: fixed

> |__General [A1]__| Optimized for better accuracy with comparable inference time as General Domain. Recommend for larger dataset/harder user scenario. This domain will use more training time. ID: `a8e3c40f-fb4a-466f-832a-5e457ae4a344`|

Reviewer: ```suggestion
|__General [A1]__| Optimized for better accuracy with comparable inference time as General Domain. Recommended for larger datasets or more difficult user scenarios. This domain requires more training time. ID: `a8e3c40f-fb4a-466f-832a-5e457ae4a344`|
```

|Domain|Purpose|
|---|---|
|__General__| Optimized for a broad range of object detection tasks. If none of the other domains are appropriate, or you are unsure of which domain to choose, select the General domain. ID: `da2e3a8a-40a5-4171-82f4-58522f70fbc1`|

> |__General [A1]__| Optimized for better accuracy with comparable inference time as General Domain. Recommend for accurate region location need/ larger dataset/harder user scenario. This domain will use more training time, and this domain is not deterministic, can expect a +-1% mAP difference with the same training data provided. ID: `9c616dff-2e7d-ea11-af59-1866da359ce6`|

Reviewer: ```suggestion
|__General [A1]__| Optimized for better accuracy with comparable inference time as General Domain. Recommended for more accurate region location needs, larger datasets, or more difficult user scenarios. This domain requires more training time, and results are not deterministic: expect a +-1% mAP difference with the same training data provided. ID: `9c616dff-2e7d-ea11-af59-1866da359ce6`|
```
